config/attribute-map.yml

```diff
@@ -0,0 +1,18 @@
+# Add your SAML attribute mapping here.
```

Comment:
lint: Minor one - unless it needs to be kebab-case for some other reason, file names should generally be snake_case.

Reply:
Okay, not an issue then. Actually working trumps style concerns!